### PR TITLE
feat: Add possibility to add some custom labels for ingress

### PR DESCRIPTION
**Pull request by pmeyer34**
related to #3294

### CODE REVIEW

File: charts/artifact-hub/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: artifact-hub
 description: Artifact Hub is a web-based application that enables finding, installing, and publishing Kubernetes packages.
 type: application
-version: 1.15.0
```
Comment (tegioz) [inline charts/artifact-hub/Chart.yaml:5]: We need to bump the version to make the Helm linter happy. Can you update it to `1.15.1-0`?

File: charts/artifact-hub/templates/hub_ingress.yaml [hunk 2]
```diff
@@ -4,6 +4,9 @@ kind: Ingress
 metadata:
   name: {{ include "chart.resourceNamePrefix" . }}hub
   labels:
+  {{- if gt (len .Values.hub.ingress.extraLabels)  0 }}
```
Comment (tegioz) [inline charts/artifact-hub/templates/hub_ingress.yaml:7]: Could you use a `with` action instead of the `if`? (for consistency across the templates)

**Review by tegioz — CHANGES_REQUESTED** [review]
Thanks @pmeyer34!

I've left a couple of minor notes, nothing important 🙂

File: charts/artifact-hub/templates/hub_ingress.yaml (hunk 2 above)
Comment (pmeyer34) [reply charts/artifact-hub/templates/hub_ingress.yaml]: Hello, done

File: charts/artifact-hub/Chart.yaml (hunk 1 above)
Comment (pmeyer34) [reply charts/artifact-hub/Chart.yaml:5]: Hello, done

**Review by tegioz — CHANGES_REQUESTED** [review]
Thanks! 🙂

Could you remove the `tmp.yaml` and `tmp2.yaml` files? Also please make sure all commits are signed off (the DCO check is not passing at the moment, maybe one of them doesn't have it).

**Comment by pmeyer34** [comment]
sorry about last commits, it's okay now

**Comment by tegioz** [comment]
No problem, thanks!

**Review by tegioz — APPROVED** [review]

**Pull request merged**